cli/src/torii/routing.rs

```diff
@@ -361,28 +357,28 @@ mod subscription {
 #[cfg(feature = "telemetry")]
 async fn handle_version(sumeragi: Arc<Sumeragi>) -> Json {
     use iroha_version::Version;
-    /*
-        TODO REDO
-        #[allow(clippy::expect_used)]
-        reply::json(
-            &wsv.blocks()
-                .last()
-                .expect("At least genesis should always exist")
-                .value()
-                .version()
-                .to_string(),
-    )
-         */
-    warp::reply::json(b"Not implemented")
+
+    let wsv = sumeragi.wsv_clone();
+
+    #[allow(clippy::expect_used)]
+    let string = wsv
+        .blocks()
+        .last()
+        .expect("At least genesis should always exist")
+        .value()
+        .version()
+        .to_string();
+    reply::json(&string)
 }
 
 #[cfg(feature = "telemetry")]
 async fn handle_metrics(sumeragi: Arc<Sumeragi>, network: Addr<IrohaNetwork>) -> Result<String> {
-    /*
-        update_metrics(&wsv, network).await?;
-        wsv.metrics.try_to_string().map_err(Error::Prometheus)
-    */
-    Ok("".to_owned())
+    sumeragi.update_metrics(network);
```

Comment:
This is no longer `async`. Have you benchmarked with and without?

Reply:
It's the metrics. What is there to benchmark? How many metrics per second we can produce?